src/node/rpc/endpoint_registry.h

```diff
@@ -626,6 +626,9 @@ namespace ccf
 
       for (const auto& [path, verb_endpoints] : fully_qualified_endpoints)
       {
+        // Special endpoint, can only be called from the node.
+        if (path == "jwt_keys/refresh")
```

Comment:
Perhaps we should add an endpoint property to disable automatic OpenAPI generation?

Reply:
@eddyashton said it's fine to hard-code for now. Should probably be discussed in a separate issue as it's related to hiding OpenAPI info when unauthenticated.